package/yast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri May 20 08:09:04 UTC 2016 - mchandras@suse.de
+
+- Split SuSEFirewall and SuSEFirewallServices classes to separate files
+  as recommended by the Yast coding style (fate#318356, gh#yast/yast-yast2#471)
```

Comment:
in fact suse changelog guide mention 67 see https://github.com/yast/yast-storage/blob/9af1ee62c9e5a23e24d60f25fa97e754da4e7274/src/modules/Storage.rb#L5558 basic hint is to not exceed dashes line above